README.md

```diff
@@ -134,6 +134,18 @@ pip install paper2remarkable
 
 This installs the ``p2r`` command line program.
 
+## Alfred
+
+Install the [Alfred workflow][workflow].
+
+You can then use `rm` command and `rmb` (for the `--blank` pages to insert blank
+pages between pages for notes) with a URL passed. The global shortcut `Alt-P`
+will send the current selection to `p2r`.
```

Comment:
Could we mention here that the default is right-aligned using the ``-r`` flag to the script? I usually don't have the menu open and it might be good to clarify.